index.html

```diff
@@ -0,0 +1,216 @@
+              width="40px"
+              height="40px"
+            />
+          </div>
+          <div id="details-comment">
+            <div id="commenter-details">
+              <h5 id="username">Yen Humbird</h5>
+              <span id="published-date-comment">1 year ago</span>
+            </div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit.
+            </p>
+            <img
+              src="./assets/imgs/comment-box-social.png"
+              alt=""
+              width="100px"
+              height="22px"
+            />
+          </div>
+        </div>
+        <div id="comment-watcher">
+          <div>
+            <img
+              class="commenter-avatar"
+              src="./assets/imgs/photo.jpg"
+              alt=""
+              width="40px"
+              height="40px"
+            />
+          </div>
+          <div id="details-comment">
+            <div id="commenter-details">
+              <h5 id="username">Nissim MV</h5>
+              <span id="published-date-comment">2 years ago</span>
+            </div>
+            <p>
+              Nice video.
+            </p>
+            <img
+              src="./assets/imgs/comment-box-social.png"
+              alt=""
+              width="100px"
+              height="22px"
+            />
+          </div>
+        </div>
+      </section>
+      <aside id="related-videos">
+        <img src="./assets/imgs/thumbmail_video_1.jpeg" alt="" />
+        <img src="./assets/imgs/thumbmail_video_2.jpeg" alt="" />
+        <img src="./assets/imgs/thumbmail_video_3.jpeg" alt="" />
+      </aside>
+    </main>
+  </body>
+</html>
+
+<!-- <div id="comment-body">
+            <div id="individual-comment">
+              <div id="comment-avatar">
+                <img src="./assets/imgs/commentusericon.png" alt="" />
+              </div>
+              <div id="indi-comment-wrapper">
+                <div id="indi-comment-1">
+                  <span>Mathiew Humbird</span>
+                  <span style="color: grey">4 years ago</span>
+                </div>
+                <div id="indi-comment-2">
+                  <p>
+                    Outstanding old-school documentary. No dramatizations, no
+                    faking, no humans doing dumb stuff, no action-movie
+                    soundtrack. Just the animals being shown in their habitat, a
+                    narrator and a soothing background music... Awesome.
+                  </p>
+                </div>
+                <div id="indi-comment-3">
+                  <div id="vote-up">
+                    <img src="./assets/imgs/comment-box-social.png" alt="" />
+                    <span style="color: grey">REPLY</span>
+                  </div>
+                </div>
+                <div id="indi-comment-4">
+                  <span>View 4 replies</span>
+                </div>
+              </div>
+            </div>
+          </div> -->
```

Comment:
Team if you have code that you don't need any more it is better to remove it. You should maintain your code clean.

Reply:
Done it, thanks.